src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaCollection/MediaCollection.js

```diff
@@ -125,6 +127,8 @@ class MediaCollection extends React.Component<Props> {
                         listStore={mediaListStore}
                         mediaListRef={mediaListRef}
                         onMediaClick={this.handleMediaClick}
+                        onUploadOverlayOpen={onUploadOverlayOpen}
```

Comment:
I personally would not add 2 params here a `uploadable=true`  doesn't make sense when we don't set `onUploadOverlayOpen` so this would conflict so I would go more then with:

```js
                        onUploadOverlayOpen={(addable && collectionStore.id) ? onUploadOverlayOpen : null}
 ```